features/admin/promotion/removing_promotions/preventing_removal_of_promotions_in_use.feature

```diff
@@ -1,4 +1,4 @@
-@managing_promotions
+@removing_promotions
```

Comment:
Is this new tag needed? I know that we have a similar tag for catalog promotion(`@removing_catalog_promotions`), but we don't have one for other resources.